build/common.mk

```diff
@@ -147,8 +148,14 @@ all:: build install lint test_all
 
 ensure::
 	$(call STEP_MESSAGE)
-	@if [ -e 'Gopkg.toml' ]; then echo "dep ensure -v"; dep ensure -v; \
```

Comment:
Are all of the providers finally off of `dep`?

Reply:
yes! This will be synced across all the providers as we add support for the upgrade